clouddriver-docker/src/main/groovy/com/netflix/spinnaker/clouddriver/docker/registry/metrics/UrlMetricsInstrumentation.groovy

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2018 Netflix, Inc.
```

Comment:
I think this should be Bol rather than Netflix

Reply:
Thanks; updated both instances.